onnxruntime/core/providers/nnapi/nnapi_builtin/model.h

```diff
@@ -102,6 +110,8 @@ class Model {
   ANeuralNetworksCompilation* compilation_{nullptr};
   ANeuralNetworksExecution* execution_{nullptr};
 
+  size_t dynamic_output_buffer_size_{1024};
+
```

Comment:
Why size_t?
How is 1024 chosen?
What happens if the buffer is too small?

Reply:
Add comments, this should be an option can be changed by user, since we cannot really predict how many elements the output will be


---
In reply to: [462102618](https://github.com/microsoft/onnxruntime/pull/4650#discussion_r462102618) [](ancestors = 462102618)